src/Popover/Popover.js

```diff
@@ -380,7 +380,7 @@ const Popover = React.createClass({
 
   render() {
     return (
-      <div>
+      <div style={{display: 'none'}}>
```

Comment:
What about moving the style object outside of the render method in a `getStyles` method?